liquidator/liquidator.js

```diff
@@ -36,29 +36,23 @@ class Liquidator {
     // Get the latest undercollateralized positions from the client.
     const underCollateralizedPositions = this.empClient.getUnderCollateralizedPositions(priceFeed);
 
-    if (underCollateralizedPositions.length > 0) {
```

Comment:
I like removing this log, I think it qualifies as too much noise. Its more helpful to know that there are no undercollateralized positions